imcsdk/apis/admin/communication_services.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016 Cisco Systems, Inc.
```

Comment:
@britthouser the code looks good. Can we not call this `communication_services.py`. On the UCSM UI, "communication services" refers to many other things. let us just call it `ipmi.py`